command/agent/config/config_test.go

```diff
@@ -278,3 +285,126 @@ func TestLoadConfigFile_AgentCache_AutoAuth_NoSink(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
+
+// TestLoadConfigFile_Template_Single tests template definitions in Vault Agent
+// configuration files
+func TestLoadConfigFile_Template(t *testing.T) {
+	testCases := map[string]struct {
+		fixturePath       string
+		expectedTemplates []*ctconfig.TemplateConfig
+	}{
+		"min": {
+			fixturePath: "./test-fixtures/config-template-min.hcl",
```

Comment:
`./testdata/...` would be better, since it is the special path for test data that the Go tool will ignore, but I realize this one was already in use so it may not be worth changing.

Reply:
If that's a go idiom, then we should adopt it, but I'll defer that to another PR